lib/readline.js

```diff
@@ -921,6 +921,9 @@ exports.Interface = Interface;
 const KEYPRESS_DECODER = Symbol('keypress-decoder');
 const ESCAPE_DECODER = Symbol('escape-decoder');
 
+let timeoutRef;
+const ESCAPE_CODE_TIMEOUT = 100;
```

Comment:
I don't know but others do. `vim` (`:h timeoutlen ` - _default_ is **1s**) , `zsh` (`$KEYTIMEOUT` - _default_ is **0.4s**, `ncurses` (`KEYDELAY` - _default_ is **1s**)